pkg/ccl/crosscluster/replicationtestutils/testutils.go

```diff
@@ -208,6 +208,10 @@ func (c *TenantStreamingClusters) ConnectToReaderTenant(
 		TenantName:  roachpb.TenantName(tenantName),
 		UseDatabase: "defaultdb",
 	})
+	c.DestSysSQL.Exec(c.T, `ALTER TENANT $1 SET CLUSTER SETTING jobs.registry.interval.adopt = '1s'`, tenantName)
+	c.DestSysSQL.Exec(c.T, `ALTER TENANT $1 SET CLUSTER SETTING bulkio.stream_ingestion.standby_read_ts_poll_interval = '500ms'`, tenantName)
+	// TODO(annezhu): speed up the ts we use for historical queries.
```

Comment:
@fqazi any tips for speeding up the rate that `watchForUpdates` gets called by the lease manager, so the historical timestamp doesn't lag too far behind? I tried bumping the usual rangefeed settings, but that didn't help. nor did `SET CLUSTER SETTING sql.catalog.descriptor_lease_duration='30s';`

Reply:
Its probably linked to the closed timestamp, we can try reducing: kv.rangefeed.closed_timestamp_refresh_interval setting. Since that determines the timestamp that we are selecting, we pick the next closed one to ensure no updates show up.

Reply:
alas, that didn't work. We'll look into this outside of this PR.